### PR TITLE
Fixes #1655 (https://github.com/facebook/react-native/issues/1655) 

### LINKED ISSUE #1655: [Navigator] SceneConfigs: PushFromRight and FloatFromRight seem to do the same type of animation
Is one deprecated?
Demo:

woot

### PR DESCRIPTION
**Pull request by neciu**
Added iOS like animation to fix the issue: https://github.com/facebook/react-native/issues/1655

### CODE REVIEW

**Comment by mkonicek** [comment]
cc @ericvicenti

File: Libraries/CustomComponents/Navigator/NavigatorSceneConfigs.js [hunk 1]
```diff
@@ -507,7 +523,10 @@ var BaseConfig = {
 var NavigatorSceneConfigs = {
   PushFromRight: {
     ...BaseConfig,
-    // We will want to customize this soon
+    animationInterpolators: {
+      into: buildStyleInterpolator(FromTheRight),
+      out: buildStyleInterpolator(ToTheLeftIOS),
```
Comment (ericvicenti) [inline Libraries/CustomComponents/Navigator/NavigatorSceneConfigs.js]: the default PushFromRight config includes `ToTheLeftIOS`, even on Android? Can we make this platform specific, or rename `ToTheLeftIOS`?

**Comment by ericvicenti** [comment]
@neciu, Can you post a video or gif of what this looks like?

**Comment by neciu** [comment]
@ericvicenti here you are:

Old one:
118fhs

New one:
118fi6

Pleas tell me what do you think about it.

**Comment by facebook-github-bot-1** [comment]
@ericvicenti would you mind taking a look at this pull request? It's been a while since the last commit was reviewed.

**Comment by grabbou** [comment]
Closing as another PR was already merged https://github.com/facebook/react-native/pull/5061

**Pull request closed without merging**